### PR TITLE
fix issue StatsWidget - padding issues #1336

### PR DESCRIPTION
**Pull request by shilomangam**
## Before you request a review, please make sure to check the following boxes:

- [x] Explain what has changed.
- [x] Explain why the change is needed.
- [x] Added tests N/A
- [x] Update documentation N/A
- [x] Merge master with no conflicts.
- [x] Make sure the build is passing on ci.
- [x] If this pr closes any issue, please add a github link to close it.
- [x] If you changed something in the product design/UX, you got the approval of [Ben Benhorin] (https://wix.slack.com/messages/@benb) N/A.

### Thanks for contributing :)

Wrap container in border-box
Center align title

[ISSUE 1336](https://github.com/wix/wix-style-react/issues/1336)

### CODE REVIEW

**Comment by shlomitc** [comment]
Please check that the 100% and flex solution works in browsers other than Chrome

File: src/Card/Content/Content.scss [hunk 1]
```diff
@@ -4,6 +4,6 @@ $content-padding: 24px 30px 18px;
 
 .content {
   @extend .container;
-
+  box-sizing: border-box;
```
Comment (shlomitc) [inline src/Card/Content/Content.scss]: remove. wrap the `StatsWidget` component with `Container` instead

File: src/Card/Header/Header.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ class Header extends WixComponent {
 
     const titleElement = (
       <div data-hook="title" className={styles.title}>
-        {title}
+        <span>{title}</span>
```
Comment (argshook) [inline src/Card/Header/Header.js]: what's `<span>` for?

Comment (shilomangam) [reply src/Card/Header/Header.js]: 1. Semantic HTML
2. Vertical alignment of the text

Comment (argshook) [reply src/Card/Header/Header.js]: 1. i see unnecessary nesting, no added semantic value
1. `flex` works well on text nodes

File: src/Card/Header/Header.scss [hunk 3]
```diff
@@ -17,6 +17,10 @@ $header-padding: 27px 30px 26px;
 
   .title {
     @extend .h2;
+    height: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
```
Comment (argshook) [inline src/Card/Header/Header.scss]: no need `flex-start` is initial

File: src/StatsWidget/StatsWidget.js [hunk 4]
```diff
@@ -49,7 +52,7 @@ class StatsWidget extends WixComponent {
       <Text dataHook="statistics-item-subtitle" appearance="H3">
         {statistics.subtitle}
       </Text>
-      { typeof (statistics.percent) !== 'undefined' && this._renderPercentage(statistics.percent)}
+      {typeof (statistics.percent) !== 'undefined' && this._renderPercentage(statistics.percent)}
```
Comment (argshook) [inline src/StatsWidget/StatsWidget.js]: this is meh, but i'd change here to `typeof statistics.percent === 'number'`

File: src/StatsWidget/StatsWidget.js [hunk 5]
```diff
@@ -62,7 +65,10 @@ class StatsWidget extends WixComponent {
 
     return (
       <Card>
-        <Card.Header dataHook="stats-widget-title" title={title} suffix={this._renderFilters(children)}/>
+        <Card.Header
+          dataHook="stats-widget-title" title={title}
```
Comment (argshook) [inline src/StatsWidget/StatsWidget.js]: `title={title}` could also go on new line

File: src/Card/Header/Header.driver.js [hunk 6]
```diff
@@ -8,7 +8,7 @@ const headerDriverFactory = ({element, wrapper, component}) => {
 
   return {
     exists: () => !!element,
-    title: () => title && title.innerHTML,
+    title: () => title && title.textContent,
```
Comment (argshook) [inline src/Card/Header/Header.driver.js]: what's the reason of this change? i hope not but it could break some builds.

if we say `textContent` is better in such cases, then `subtitle` should also use that

Comment (shilomangam) [reply src/Card/Header/Header.driver.js]: TextContent is always better, we're testing the behaviour and not the implementation. thats not suppose to break any build. i will change it also in subtitle, thanks!
